lib/features/pfis/pfis_service.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:didpay/features/pfis/pfi.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:web5/web5.dart';
+
+final pfisServiceProvider = Provider((_) => PfisService());
+
+class PfisService {
+  Future<Pfi> createPfi(String input) async {
+    final did = _parseDid(input);
+    final res = await _resolveDid(did);
+    final didDocument = _getDidDocument(res);
```

Comment:
any reason to split all these out into separate functions? do you foresee us re-using these anywhere aside from right here?

Reply:
nahh i was just braindead haha